data/json/items/containers.json

```diff
@@ -175,7 +175,7 @@
     "type": "GENERIC",
     "category": "container",
     "name": { "str": "garbage bag" },
-    "looks_like": "bag_canvas",
+    "looks_like": "bag_garbage",
```

Comment:
Why are you changing the "looks_like" of garbage bags to look like themselves? That's not what "looks_like" is for; it's meant as a backup if there aren't sprites made for it yet. It was set to a canvas bag to tell the game "If there aren't sprites for the garbage bag, use the sprites for canvas bags as a backup." By changing it to garbage bag, you're telling the game "If there aren't sprites for the garbage bag, use the sprites for garbage bags as a backup" which would just lead to missing textures.